conanfile.py

```diff
@@ -8,7 +8,7 @@
 
 class SISLConan(ConanFile):
     name = "sisl"
-    version = "8.5.4"
+    version = "8.5.5"
```

Comment:
This should really be `9.x` since we are removing interfaces, but since we are committed to continuing with `8.x` can we at least bump the major to `8.6.0`?

Reply:
Ack

Reply:
Ack